tests/epics/test_signals.py

```diff
@@ -360,8 +358,15 @@ async def test_pvi_structure(ioc: IOC) -> None:
             "height": {
                 "rw": f"{PV_PREFIX}:{ioc.protocol}:height",
             },
-        }
+        },
+        "record": {
+            "_options": {
+                "atomic": ANY,  # the field can be true or false
+                "queueSize": 0,
+            },
+        },
```

Comment:
Is the atomic not deterministic from the dbrecord?

I guess it's only the pvi entry that is used, I'm just trying to understand

Reply:
It's appended when you actually read the pva table:
```
pvinfo PREFIX:SEQ1:PVI
PREFIX:SEQ1:PVI
Server: 172.23.244.139:5075
Type:
    structure
        structure record
            structure _options
                int queueSize
                boolean atomic
        structure pvi
            structure active
                string r
            structure bita
                string rw
            structure bita_delay
```

`_atomic` differs between using the `backend.get_reading()` and `backend.update()`:
```
backend reading True
update value False
update reading False
backend value True
```